tests/unittests/StatsExporterTest.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/Runtime/StatsExporter.h"
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+using namespace glow;
+
+using testing::ElementsAre;
+
+class MockStatsExporter : public StatsExporter {
+public:
+  MockStatsExporter() { StatsExporter::registerStatsExporter(this); }
+
+  ~MockStatsExporter() override {}
+
+  void addQuantileValue(llvm::StringRef key, double value) override {
```

Comment:
quantiles seems a bit niche and might not be supported by every exporter, maybe it should be an optional feature for an exporter?

Reply:
Hm, interesting point.  What I expect this type of counter to be used for is something where an aggregated timeseries makes sense (so, like, p95 queue latency over a 60 second window), as opposed to the other counter, which is more of a static value.  This counter type doesn't necessarily have to specifically use quantiles (although I'm planning to hook up fb303's quantile stats to it).

Maybe it should be more generically named like `addTimeseriesValue`?

Hm, also it occurs to me this class design is kind of bad in that anything inheriting from the base StatsExporter without overriding is going to get the "iterate the registered list" behavior, which is unintuitive.  I'll fix it so these methods are actually optional.